docs/content/users/configuration/config.md

```diff
@@ -518,10 +518,12 @@ The DDEV-specific project type.
 
 | Type | Default | Usage
 | -- | -- | --
-| :octicons-file-directory-16: project | `php` | Can be `backdrop`, `craftcms`, `django4`, `drupal6`, `drupal7`, `drupal8`, `drupal9`, `drupal10`,  `laravel`, `magento`, `magento2`, `php`, `python`, `shopware6`, `silverstripe`, `typo3`, or `wordpress`.
+| :octicons-file-directory-16: project | `php` | Can be `backdrop`, `craftcms`, `django4`, `drupal6`, `drupal7`, `drupal`,  `laravel`, `magento`, `magento2`, `php`, `python`, `shopware6`, `silverstripe`, `typo3`, or `wordpress`.
 
 The `php` and `python` types don’t attempt [CMS configuration](../../users/quickstart.md) or settings file management and can work with any project.
 
+The legacy `drupal8`, `drupal9`, and `drupal10` project types can also be specified, but if you just use `drupal` DDEV will autodetect the type and use the recommended settings.
```

Comment:
```suggestion
The legacy `drupal8`, `drupal9`, and `drupal10` project types can also be specified, but by using `drupal` DDEV will autodetect the type and use the recommended settings.
```

Reply:
> The legacy `drupal8`, `drupal9`, and `drupal10`

There has been talk about using the terms "modern drupal" for Drupal 8+, and "legacy Drupal" for Drupal =<7. 
With that in mind, I think we should avoid using "legacy" in this doc context.

Reply:
Good point @tyler36 - They're "legacy" only in the sense that we're not going to use specific terms in DDEV any more.  Please suggest alternative wording, thanks.

Reply:
How about the following?  
`The project types drupal8, drupal9, and drupal10 can still be directly specified,...`

Reply:
> The project types drupal8, drupal9, and drupal10 can still be directly specified,...

@rpkoller, I like that! It keeps with the DDEV terminology.

Reply:
Updated to 
> The former DDEV project types `drupal8`, `drupal9`, and `drupal10` can still be manually specified; by using `drupal` instead, DDEV will autodetect the correct type and its corresponding settings.